test/scripts/jenkins_report_failed_tests.sh

```diff
@@ -1,5 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Will create a PR to remove all references to this from Jenkins - but we should be OK to remove now since the task isn't actually running.